types/stripe/index.d.ts

```diff
@@ -4073,12 +4073,12 @@ declare namespace Stripe {
 
     namespace bankAccounts {
         interface IBankAccount extends IBankAccountHash {
-            account: string;
+            account?: string;
 
             /**
              * This indicates whether or not this bank account is the default external account for its currency.
              */
-            default_for_currency: boolean;
+            default_for_currency?: boolean;
```

Comment:
This does not appear to be present in most IBankAccount responses, and it is not mentioned in the docs at https://stripe.com/docs/api/customer_bank_accounts/object